Add ability to specify heatmap weights.

frslm opened the pull request:
To address #34.

sandrotosi commented on gmplot/gmplot.py:
```diff
@@ -169,9 +177,12 @@ def heatmap(self, lats, lngs, threshold=10, radius=10, gradient=None, opacity=0.
         settings['dissipating'] = dissipating
         settings = self._process_heatmap_kwargs(settings)
 
+        if weights is None:
```
probably `if not weights`? if people pass `[]` (ok that'd be weird) the `zip()` call later on will return no results

maybe we should also introduce checks that `len(lats) == len(lngs) == len(weights)` ?

frslm replied:
Hmm, I was actually trying to avoid a case where people pass in `0`...

I think we can leave that as is, but yea we could replace all `zip()` calls with a wrapper function that ensures the length of all `zip()` arguments are equal to each other. If that works for you, I can create a new issue and implement this wrapper function there.

sandrotosi commented on gmplot/gmplot.py:
```diff
@@ -429,9 +440,12 @@ def write_polygon(self, f, path, settings):
-                        (heatmap_lat, heatmap_lng))
+            for lat, lng, weight in heatmap_points:
+                location = 'new google.maps.LatLng(%f, %f)' % (lat, lng)
+                if weight == self.HEATMAP_DEFAULT_WEIGHT:
```
should we just write out the weight regardless, even if it's just the default?

frslm replied:
I don't think so; it seems like that'll just inflate the size of the HTML file for no gain.

frslm commented:
Manually merged since this has been open for some time and there weren't any unaddressed comments. If anything should be addressed, we can tackle it in a separate issue.
